spec/history_spec.rb

```diff
@@ -50,7 +50,7 @@ def stub_hist(options)
 
     context "when $XDG_DATA_HOME is defined" do
       it "returns config location relative to $XDG_DATA_HOME" do
-        stub_hist has_default: false, xdg_home: '/my/path'
+        stub_hist has_default: true, xdg_home: '/my/path'
```

Comment:
We shouldn't be modifying the existing test because we want to test all scenarios. With this change we no longer test the situation when `~/.pryrc` doesn't exist but the XDG-compatible config does.

I would add a separate with `stub_hist has_default: true, xdg_home: '/my/path'`.